player/lua/stats.lua

```diff
@@ -641,13 +643,26 @@ local function add_header(s)
 end
 
 
-local function add_file(s, print_cache)
+local function add_file(s, print_cache, print_tags)
     append(s, "", {prefix="File:", nl="", indent=""})
     append_property(s, "filename", {prefix_sep="", nl="", indent=""})
     if mp.get_property_osd("filename") ~= mp.get_property_osd("media-title") then
         append_property(s, "media-title", {prefix="Title:"})
     end
 
+    if print_tags then
+        local tags = mp.get_property_native("display-tags")
```

Comment:
Would be nice to update documentation of `display-tags` that it affects olso stats.lua now.